### ISSUE TITLE
[no_more_watering] Error in main menu

### ISSUE THREAD

**Issue by axi92** [devopsdinosaur/sunhaven-mods#11]
This error keeps posting in the log every second.
If I remove no_more_watering I have no errors.

```
[Message:   BepInEx] BepInEx 5.4.21.0 - Sun Haven (22.03.2023 13:24:14)
[Info   :   BepInEx] Running under Unity v2019.3.15.5898046
[Info   :   BepInEx] CLR runtime version: 4.0.30319.42000
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Bits64, Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.21.0]
[Info   :   BepInEx] 1 patcher plugin loaded
[Info   :   BepInEx] Patching [UnityEngine.CoreModule] with [BepInEx.Chainloader]
[Message:   BepInEx] Preloader finished
[Warning:   BepInEx] Config value of setting "Chainloader.HideManagerGameObject" could not be parsed and will be ignored. Reason: String was not recognized as a valid Boolean.; Value: flase
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 11 plugins to load
[Info   :   BepInEx] Loading [Movement Speed 0.2.1]
[Info   :   BepInEx] Loading [Configuration Manager 18.0]
[Info   :   BepInEx] Loading [Craft From Storage 0.0.9]
[Info   :Craft From Storage] devopsdinosaur.sunhaven.craft_from_storage v0.0.9 loaded.
[Info   :   BepInEx] Loading [Instant Text 0.0.1]
[Info   :Instant Text] devopsdinosaur.sunhaven.instant_text v0.0.1 loaded.
[Info   :   BepInEx] Loading [No More Deadlines 0.0.1]
[Info   :No More Deadlines] devopsdinosaur.sunhaven.no_more_deadlines v0.0.1 loaded.
[Info   :   BepInEx] Loading [No More Watering 0.0.5]
[Info   :No More Watering] devopsdinosaur.sunhaven.no_more_watering v0.0.5 loaded.
[Info   :   BepInEx] Loading [Skill Reset 0.0.1]
[Info   :Skill Reset] devopsdinosaur.sunhaven.skill_reset v0.0.1 loaded.
[Info   :   BepInEx] Loading [Stack Size 0.0.1]
[Info   :Stack Size] devopsdinosaur.sunhaven.stack_size v0.0.1 loaded.
[Info   :   BepInEx] Loading [Time Management 0.0.3]
[Info   :Time Management] devopsdinosaur.sunhaven.time_management v0.0.3 loaded.
[Info   :   BepInEx] Loading [KeepAlive 0.0.1]
[Warning:       Log] Plugin KeepAlive is loaded!
[Info   :   BepInEx] Loading [No Time For Fishing! 0.0.3]
[Warning:       Log] Plugin No Time For Fishing! is loaded!
[Message:   BepInEx] Chainloader startup complete
[Info   : Unity Log] 4246
[Info   : Unity Log] Application Persistent Data Path: C:/Users/User/AppData/LocalLow/Pixel Sprout Studios/Sun Haven
OS:Windows 10  (10.0.0) 64bit
Setting breakpad minidump AppID = 1432860
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  76561198037559430 [API loaded no]
[Info   : Unity Log] Load Char
[Warning: Unity Log] DontDestroyOnLoad only works for root GameObjects or components on root GameObjects.
[Info   : Unity Log] Making player
[Info   : Unity Log] Load Char
[Info   : Unity Log] Mirror | mirror-networking.com | discord.gg/N9QVxbM
[Info   : Unity Log] Update Quest Items
[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Plugin+HarmonyPatch_Player_Update.Prefix (Wish.Player& __instance) (at <63e616b204d24fb7bfdca1eebe0f69a8>:0)
(wrapper dynamic-method) Wish.Player.DMD<Wish.Player::Update>(Wish.Player)

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Plugin+HarmonyPatch_Player_Update.Prefix (Wish.Player& __instance) (at <63e616b204d24fb7bfdca1eebe0f69a8>:0)
(wrapper dynamic-method) Wish.Player.DMD<Wish.Player::Update>(Wish.Player)

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Plugin+HarmonyPatch_Player_Update.Prefix (Wish.Player& __instance) (at <63e616b204d24fb7bfdca1eebe0f69a8>:0)
(wrapper dynamic-method) Wish.Player.DMD<Wish.Player::Update>(Wish.Player)

[Error  : Unity Log] NullReferenceException: Object reference not set to an instance of an object
Stack trace:
Plugin+HarmonyPatch_Player_Update.Prefix (Wish.Player& __instance) (at <63e616b204d24fb7bfdca1eebe0f69a8>:0)
(wrapper dynamic-method) Wish.Player.DMD<Wish.Player::Update>(Wish.Player)
```

**Comment by devopsdinosaur** [comment]
Just uploaded v0.0.6 that should eliminate this issue.  Let me know if you continue to see it.

**Comment by axi92** [comment]
Error is fixed, but plugin shows 0.0.5

```
[Info   :   BepInEx] Loading [No More Watering 0.0.5]
[Info   :No More Watering] devopsdinosaur.sunhaven.no_more_watering v0.0.5 loaded.
```

Thats the filehash:
```
user@DESKTOP:/mnt/e/Steam/steamapps/common/Sun Haven/BepInEx/plugins$ sha256sum devopsdinosaur.sunhaven.no_more_watering.dll
9bafbb4cecea20032f915d26ecf09bf72ce6d1a99b4bd3a077a3e5e323b1a69c  devopsdinosaur.sunhaven.no_more_watering.dll
```

**Comment by devopsdinosaur** [comment]
You are exactly right.  I forgot to update the version in the source.  I uploaded a new mod that prints the correct version.  Thanks for the heads up.
